pallets/pools/src/lib.rs

```diff
@@ -391,8 +426,12 @@ pub mod pallet {
 
 	#[pallet::storage]
 	#[pallet::getter(fn scheduled_update)]
-	pub type ScheduledUpdate<T: Config> =
-		StorageMap<_, Blake2_128Concat, T::PoolId, ScheduledUpdateDetails<T::InterestRate>>;
+	pub type ScheduledUpdate<T: Config> = StorageMap<
+		_,
+		Blake2_128Concat,
+		T::PoolId,
+		ScheduledUpdateDetails<T::InterestRate, T::MaxTokenNameLength, T::MaxTokenSymbolLength>,
```

Comment:
This will actually be a migration. There is no migration available, right? I am pretty sure, we do not have any scheduled updates in place. 

@branan and @offerijns , we could go without writing a migration here. I am a little hesitant, but think it would be okayish.

Reply:
I think it's fine to not have a migration for this.